app/views/application_searches/show.html.erb

```diff
@@ -18,5 +18,31 @@
     </div>
   </div>
 
-  <%= render 'search_results_table', results: @search.results, form_params: { sorting: @sorting, filter: @filter, url: application_searches_path }  %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <table class="govuk-table app-dashboard-table" data-module="moj-sortable-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <%= table_headers(
```

Comment:
Why not use the partial like in `app/views/application_searches/_search_results_table.html.erb`?